src/Toolkit/A.php

```diff
@@ -726,4 +726,35 @@ public static function wrap($array = null): array
             return $array;
         }
     }
+
+    /**
+     * Filter the array using the given callback
+     * using both value and key
+     *
+     * @param array $array
+     * @param callable $callback
+     * @return array
+     */
+    public static function filter(array $array, callable $callback): array
+    {
+        return array_filter($array, $callback, ARRAY_FILTER_USE_BOTH);
+    }
+
+    /**
+     * Remove key(s) from an array
+     *
+     * @param array $array
+     * @param int|string|array $keys
```

Comment:
Super tiny nitpick: I think this argument should be called `$key` for consistency with `A::get()`.

Reply:
@lukasbestle yes, but `$collection->without()` argument is called `$keys` 👀